SpecflowTests/Bind_Steps/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,214 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using TechTalk.SpecFlow;
+using Onboarding_Task_Mars_Specflow.SpecflowPages.Helpers;
+using Onboarding_Task_Mars_Specflow.SpecflowPages.Pages;
+
+namespace Onboarding_Task_Mars_Specflow.SpecflowTests.Bind_Steps
+{
+    [Binding]
+    public class LanguageStepDefinitions : DriverHelpers
+    {
+        private WebDriverWait _wait;
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors